composer.json

```diff
@@ -16,14 +16,14 @@
     ],
     "require": {
         "php": "^7.3|^8.0",
-        "tipoff/authorization": "^2.7.1",
-        "tipoff/bookings": "^2.3.0",
-        "tipoff/escape-room": "^2.2.3",
-        "tipoff/locations": "^2.8.2",
-        "tipoff/support": "^1.8.9"
+        "tipoff/authorization": "^2.8.2",
+        "tipoff/bookings": "^2.5.1",
+        "tipoff/escape-room": "^2.3.0",
```

Comment:
@drewroberts I'm not quite sure what happened, but updating my dependencies with these new package versions is throwing errors everything. I'm experiencing the following error repeatedly:

```
Caused by
PDOException: SQLSTATE[HY000]: General error: 1 no such table: images

/Users/chrishackett/Code/packages/feedback/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:79
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Connection.php:338
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Connection.php:671
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Connection.php:638
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Connection.php:346
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2313
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2301
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2796
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2302
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2723
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2651
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:1547
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Support/Traits/ForwardsCalls.php:23
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1890
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1902
/Users/chrishackett/Code/packages/feedback/vendor/tipoff/support/src/Helpers/Test.php:31
/Users/chrishackett/Code/packages/feedback/vendor/drewroberts/blog/database/factories/PageFactory.php:30
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:385
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:364
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:348
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Concerns/GuardsAttributes.php:151
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:353
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:321
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:233
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:230
/Users/chrishackett/Code/packages/feedback/vendor/tipoff/locations/database/factories/LocationFactory.php:26
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:385
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:364
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:348
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Concerns/GuardsAttributes.php:157
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:353
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:321
/Users/chrishackett/Code/packages/feedback/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Factories/Factory.php:233
/Users/chrishackett/Code/packages/feedback/tests/Unit/Mail/FeedbackSummaryTest.php:25
```

Reply:
Ahh, it looks like we need to import more package service providers to the test case